radio/src/targets/common/arm/stm32/stm32_spi.cpp

```diff
@@ -271,6 +286,17 @@ void stm32_spi_set_max_baudrate(const stm32_spi_t* spi, uint32_t baudrate)
 #endif
 }
 
+#if defined(STM32H7) || defined(STM32H7RS)
+void stm32_spi_set_data_width(const stm32_spi_t* spi, uint32_t dataWidth)
+{
+  auto* SPIx = spi->SPIx;
+  LL_SPI_Disable(SPIx);
+  LL_SPI_SetDataWidth(SPIx, dataWidth);
+  LL_SPI_Enable(SPIx);
+  LL_SPI_StartMasterTransfer(SPIx);
```

Comment:
Why start a transfer at this point? This feels odd. Besides, this function is not specific to H7 and could be defined for other targets as well, right?